pages/common/badblocks.md

```diff
@@ -14,3 +14,15 @@
 - Search an unmounted disk for bad blocks with a destructive write test:
 
 `sudo badblocks -w {{/dev/sda}}`
+
+- Search an unmounted disk for bad blocks with a destructive write test and show verbose status:
+
+`sudo badblocks -svw {{/dev/sda}}`
+
+- Search an unmounted disk in desctructive mode and output found blocks to a file:
```

Comment:
Typo at destructive.

Also, "output" is not a verb. Let's change it to "write"